scripts/dream.py

```diff
@@ -1,16 +1,15 @@
 #!/usr/bin/env python3
```

Comment:
```sh
$ scripts/dream.py
bash: scripts/dream.py: Permission denied
```
This PR breaks running dream.py as a script because it flips off the executable bit, see the pull request, specifically `100755 → 100644` in the title for the `scripts/dream.py`.